build-system/tasks/coverage-map/index.js

```diff
@@ -93,9 +95,11 @@ async function coverageMap() {
   puppeteer = require('puppeteer');
   explore = require('source-map-explorer').explore;
 
-  await dist();
+  if (!argv.nodist) {
```

Comment:
can we be consistent, i believe on the other commands it will be `nobuild`?

Reply:
Done.